Scripts/Morphology_main.py

```diff
@@ -63,70 +54,95 @@ def default(self, obj):
             return obj.tolist()
         return json.JSONEncoder.default(self, obj)
 
-def main(input_file, metadata_file, output_measure, output_landmark, output_presence, 
-         output_lm_image=None):
+def argument_parser():
+    parser = argparse.ArgumentParser(description='Extract information from segmented fish image such as presence absence,\
+                                     landmarks, measures.')
+    parser.add_argument('input_image', help='Path of segmented fish image. Format PNG image file.')
+    parser.add_argument('output_presence', help='Path of output presence absence table. Format JSON file.')
+    
+    
+    parser.add_argument('--metadata', help='Path of input drexel_metadata_formatter. Format JSON metadata file.')
+    parser.add_argument('--morphology', 
+                        help='Save the dictionnary of morphology measurements with the provided filename.')
+    parser.add_argument('--landmark', 
+                        help='Save the dictionnary of landmarks with the provided filename.')
+    parser.add_argument('--lm_image', 
+                        help='Save the visualisation of landmarks with the provided filename.')
+    return parser
+
+def main():
+    '''
+    Use Class Segmented_image, Measure_morphology to extract information 
+    from a segmented fish image (.png image).
+    
+    Input output are managed by argument_parser()
+    
+    Segmented_image creates an object that isolates different traits, preprocesses them and extract genral info (such as fish angle)
+    Measure_morphology inherits from Segmented_image and uses preprocessed traits information to measure morphology
+    characteristic and landmarks.
+    There are 4 mains output defined in argument_parser()
+    args.presence : {"dorsal_fin": {"number": 1, "percentage": 1.0}, "adipos_fin": {"number": 0, "percentage": 0}....}
```

Comment:
I think this is now named `output_presence`?

Reply:
fixed 
https://github.com/hdr-bgnn/Morphology-analysis/commit/c718c28fbbac5edf31d85b437d31a877d84c6afc